src/components/projectSelect/index.tsx

```diff
@@ -83,3 +111,23 @@ export const ProjectSelect: React.FC<ProjectSelectProps> = ({
     />
   );
 };
+
+export const getRepoIdentifier = (projects) => {
+  if (!projects || projects.length === 0) {
+    return "";
+  }
```

Comment:
I'm not sure this case needs to be handled. This shouldn't be called if projects doesn't exist.

Reply:
True, but I don't want to take the risk of accessing an array index without knowing with certainty that it exists.

Reply:
fair enough but in that case maybe we should error?

Reply:
I can, but I don't know that it is necessary, because the user wouldn't know what to do with an error like that.

Reply:
perfect, I like that better than an error toast to the user. I'll do that.

Reply:
just added this